README.md

```diff
@@ -254,36 +257,61 @@ For more complex integrations, you may pass a `getLicense` function to fully con
 }
 ```
 
+
+#### Get Certificate by function
 Although the license acquisition is the only required configuration, `getCertificate()` is also supported if your source needs to retrieve a certificate, similar to the [FairPlay](#fairplay) implementation above.
 
-The `audioContentType` and `videoContentType` properties for non-FairPlay sources are used to determine if the system supports that codec and to create an appropriate `keySystemAccess` object. If left out, it is possible that the system will create a `keySystemAccess` object for the given key system, but will not be able to play the source due to the browser's inability to use that codec.
+Example:
+```js
+{
+  'org.w3.clearkey': {
+    getCertificate: function(emeOptions, callback) {
+      // request certificate
+      // if err, callback(err)
+      // if success, callback(null, certificate)
+    },
+  }
+}
+```
+
 
-Below is an example of using one of these DRM systems and custom `getLicense()` and `getCertificate()` functions:
+### audioContentType/videoContentType
 
+The `audioContentType` and `videoContentType` properties for non-FairPlay sources are used to determine if the system supports that codec and to create an appropriate `keySystemAccess` object. If left out, it is possible that the system will create a `keySystemAccess` object for the given key system, but will not be able to play the source due to the browser's inability to use that codec.
+
+example:
 ```js
 {
   keySystems: {
     'org.w3.clearkey': {
       audioContentType: 'audio/webm; codecs="vorbis"',
       videoContentType: 'video/webm; codecs="vp9"',
-      getCertificate: function(emeOptions, callback) {
-        // request certificate
-        // if err, callback(err)
-        // if success, callback(null, certificate)
-      },
-      getLicense: function(emeOptions, keyMessage, callback) {
-        // request license
-        // if err, callback(err)
-        // if success, callback(null, license)
-      }
+    }
+  }
+}
+```
+
+### audioRobustness/videoRobustness
+> If `audioRobustness`/`videoRobustness` is not passed in for widevine, you will see a warning similar to: `It is recommended that a robustness level be specified. Not specifying the robustness level could result in unexpected behavior`
+
+The `audioRobustness` and `videoRobustness` properties for non-FairPlay sources are used to determine the robustness level of the DRM you are using.
```

Comment:
Can we add something like:
"possible options for widevine include: `SW_SECURE_CRYPTO, SW_SECURE_DECODE, HW_SECURE_CRYPTO, HW_SECURE_DECODE, HW_SECURE_ALL`"

Reply:
Sure I will add that.